### PR TITLE
CC-686 Added LIVE Terraform config vars

### PR DESCRIPTION
**Pull request by mwestacott**
https://companieshouse.atlassian.net/browse/CC-686

* LIVE config added
* Minor change to Staging config so that CDN Host definition now correct (matches format in CiDev)

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 1]
```diff
@@ -1 +1,25 @@
-# placeholder file
\ No newline at end of file
+environment = "live"
+aws_profile = "live-eu-west-2"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
```
Comment (dlloyd1-ch) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:5]: 'live" ?

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+# service configs
+api_local_url = "https://private-api.companieshouse.gov.uk"
+cdn_host = "//db2062spq951s.cloudfront.net"
+chs_url = "https://find-and-update.company-information.service.gov.uk"
+cookie_domain = "company-information.service.gov.uk"
+piwik_site_id = "10"
+piwik_url = "https://matomo.companieshouse.gov.uk/"
+session_create = "off"
```
Comment (markpit) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:14]: is the session_create definitely 'off' in live?

Comment (mwestacott) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:14]: Yes. Confirmed by config sent to me by Darran Cornelius.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars (hunk 1 above)
Comment (mwestacott) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:5]: Ah yes 👍 will change.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars (hunk 2 above)
Comment (markpit) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:14]: ok, i also went through the release notes and can see it was set to off in first release and never changed, so should be good.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars (hunk 1 above)
Comment (mwestacott) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars]: Done.

**Review by markpit — APPROVED** [review]

**Review by dlloyd1-ch — APPROVED** [review]

**Pull request merged**